etebase_server/settings.py

```diff
@@ -130,6 +130,11 @@
 MEDIA_ROOT = os.environ.get("DJANGO_MEDIA_ROOT", os.path.join(BASE_DIR, "media"))
 MEDIA_URL = "/user-media/"
 
+ETEBASE_API_PERMISSIONS = ["rest_framework.permissions.IsAuthenticated"]
```

Comment:
These two settings are wrong, we removed them in master. Are they even needed for you, or just a merge mistake?

Reply:
Yes, they were a merge mistake.